pkg/runtime/interfaces/application_configuration.go

```diff
@@ -146,6 +146,18 @@ type NotificationsEmailerConfig struct {
 	Body string `json:"body"`
 }
 
+// This section handles configuration for the workflow notifications pipeline.
+type EventsPublisherConfig struct {
+	// The topic which events should be published, e.g. node, task, workflow
+	TopicName string `json:"topicName"`
+	// Event types: task, node, workflow executions
+	EventTypes string `json:"eventTypes"`
```

Comment:
why not an array type?

Reply:
Because I want to support the option to use `all` or `*` and using a single string makes it easier to filter on such cases.

Reply:
but you can still do that with a single entry, no? I'd prefer to use conventional yaml list formatting vs our own string-split scheme